From: Mario Sanchez Prada <mario@endlessm.com>
Date: Mon, 16 Apr 2018 05:47:57 -0500
Subject: popupMenu: Fix wrong call to clutter_actor_add_child()

Specify the horizontal alignment via the x_align property when creating
the StIcon, since this function expects one argument, not two.

Origin: https://gitlab.gnome.org/GNOME/gnome-shell/commit/cdbc99e
---
 js/ui/popupMenu.js | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/js/ui/popupMenu.js b/js/ui/popupMenu.js
index 67b928c..83194d7 100644
--- a/js/ui/popupMenu.js
+++ b/js/ui/popupMenu.js
@@ -394,8 +394,9 @@ var PopupImageMenuItem = new Lang.Class({
     _init(text, icon, params) {
         this.parent(params);
 
-        this._icon = new St.Icon({ style_class: 'popup-menu-icon' });
-        this.actor.add_child(this._icon, { align: St.Align.END });
+        this._icon = new St.Icon({ style_class: 'popup-menu-icon',
+                                   x_align: Clutter.ActorAlign.END });
+        this.actor.add_child(this._icon);
         this.label = new St.Label({ text: text });
         this.actor.add_child(this.label);
         this.actor.label_actor = this.label;
